### PR TITLE
Deploying new Staking & Escrow to OP

### PR DESCRIPTION
**Pull request by kirilantevski**
- deploying to OP for testing

### CODE REVIEW

File: scripts/deployEscrowAndStaking/deploy_transparent.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const fs = require('fs');
 const { getTargetAddress, setTargetAddress, encodeCall } = require('../helpers');
 
 const user_key1 = process.env.PRIVATE_KEY;
+const ISSUER_ADDR = '0x42d9ac3ebebb9479f24360847350b4F7EADECE50';
```
Comment (danijelthales) [inline scripts/deployEscrowAndStaking/deploy_transparent.js:19]: cant we get this from SNX library like we get proxySUSD and SafeMath?

Comment (danijelthales) [reply scripts/deployEscrowAndStaking/deploy_transparent.js:19]: its a different address per chain

File: scripts/deployments.json [hunk 2]
```diff
@@ -1,138 +1 @@
...
...35BB630Ba58cF48dfDf2E4","ProxyEscrowThales":"0x6Bd555E78cBb749FDeC771F7dc366c5Cb36d15E1","ProxyStakingThalesImplementation":"0x330EB3487D68AEd22061335D6B6af08E85068010","ProxyEscrowThalesImplementation":"0x05c0aF5Efc111a0C6A942711b37A82A00587745A","ThalesRoyaleImplementation":"0x95d97c6116C3a1d27b25C9448FDF3889d6dd7c05","ThalesRoyaleDeployed":"0xd885521B35F3DFbc128B38f8897109eba829472e","SafeBox":"0x8314125c8b68af2afd0d151eb4a551e88128a2ae"},"localhost":{"Airdrop":"","OngoingAirdrop":"0xB74DD224F1652672Fc5d18404Fe5AF120e1e0991","VestingEscrow":"0xE04Fa48Bf6f15E8Ff1317Cd8b896408235A08Df7","EscrowThales":"0x1817e45A781f47b89F5CD498E737Ee96cF743be9","StakingThales":"","Thales":"0x9E545E3C0baAB3E08CdfD552C960A1050f373042","PriceFeed":"0xa25816b9605009aa446d4d597F0AA46FD828f056","BinaryOptionMastercopy":"","BinaryOptionMarketMastercopy":"","BinaryOptionMarketFactory":"","BinaryOptionMarketManager":"","BinaryOptionMarketData":"","OpThales_L1":"0x4A679253410272dd5232B3Ff7cF5dbB88f295319","ThalesExchanger":"0x7a2088a1bFc9d81c55368AE168C2C02570cB814F","OwnedUpgradeabilityProxy":"0x1613beB3B2C4f22Ee086B2b38C1476A3cE7f78E8","ProxyThalesExchanger":"0x59b670e9fA9D0A427751Af201D676719a970857b"}}
```
Comment (danijelthales) [inline scripts/deployments.json:1]: please format

**Review by danijelthales — CHANGES_REQUESTED** [review]

File: scripts/deployments.json (hunk 2 above)
Comment (kirilantevski) [reply scripts/deployments.json:1]: done

File: scripts/deployEscrowAndStaking/deploy_transparent.js (hunk 1 above)
Comment (kirilantevski) [reply scripts/deployEscrowAndStaking/deploy_transparent.js:19]: done

**Comment by grujawork** [comment]
image

wow! 😆  i needed to check what it is 😆

**Review by danijelthales — APPROVED** [review]

**Pull request merged**